site/content/blog/2023-07-06-zap-2.13.0/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "ZAP 2.13.0"
+summary: "ZAP 2.13.0 has just been [released](/docs/desktop/releases/2.13.0/), and adds support for HTTP/2, improved authentication handling and Mac Silicon."
```

Comment:
Thought H2 was in 2.12?

Reply:
Forget it, I guess we had still said it was coming: https://www.zaproxy.org/blog/2022-10-27-zap-2-12-0-the-ten-thousand-star-release/#network-add-on